packages/UI/src/components/organisms/cards/ShareCard/index.ts

```diff
@@ -0,0 +1,8 @@
+export { default } from './ShareCard';
+
+export interface ShareCardProps {
+  title: 'idea' | 'position' | 'product';
+  platforms: string;
+  link: string;
+  handleClose: () => void;
```

Comment:
@alston-hsu 
This prop isn't used right? if so let's remove it

Reply:
Done